submissions/adeyemiJuwon/HTMLforms/contact.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>contact</title>
+</head>
+<body>
+  <header>
+    <div>
+        <input type="search" name="q" value="" aria-label="Search" placeholder="Search">
+            <ul>
+                <li>
+                    <a href="login.html">Log in</a>
+                </li>
+                <li>
+                    <a href="account.html">Sign Up</a>
+                </li>
+                <li>
+                    <a href="contact.html">Contact</a>
+                </li>
+                <li>
+                    <a href="#">Cart</a>
+                </li>
+            </ul>
+    </div>
+    <div>
+        <a href="index.html">
+            <img src="img/mlogo.png" alt="">
+        </a>
+    </div>
+    <hr>
+    <nav>
+        <ul>
+            <li><a href="#">Best Sellers</a></li>
+            <li><a href="#">Baby Care &amp; Wear</a></li>
+            <li><a href="#">Women collection</a></li>
+            <li><a href="#">Electronic Accessiories</a></li>
+            <li><a href="#"> Lights &amp; Lighting</a></li>
+            <li><a href="#">Smart Home &amp; Garden</a></li>
+
+        </ul>
+    </nav>
+</header>
+    <hr>
+    <section>
+      <h1>Contact Us</h1>
+      <p>Address: Martha's Living products Ltd.
+        43 Longley road, Tooting, London, SW17 9LA, United Kingdom</p>
+    <li> Tel number:<a href=" tell:+2348084624301"> +2348084624301</a></li>
+    <br>
+    <li>Email: <a href=" mailto:shopify.raymart7@gmail.com"> shopify.raymart7@gmail.com</a></li>
+    <br>
+    
+  <form>
+    <h3>Fill this form</h3>
+    <label for="fullname">Full Name</label><br>
+    <input type="text" id="fullname" name="fullname" placeholder="Adeyemi Juwon Samuel" required><br>
```

Comment:
Update the label for, input id & name to use camelCase or camel-case i.e `fullName` or `full-name`

Reply:
> Update the label for, input id & name to use camelCase or camel-case i.e `fullName` or `full-name`

done